great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -806,6 +867,72 @@ def inner_func(
 
         return wrapper
 
+    elif issubclass(engine, SparkDFExecutionEngine):
+        if partial_fn_type is None:
+            partial_fn_type = MetricPartialFunctionTypes.MAP_CONDITION_FN
+        partial_fn_type = MetricPartialFunctionTypes(partial_fn_type)
+        if partial_fn_type not in [
+            MetricPartialFunctionTypes.MAP_CONDITION_FN,
+            MetricPartialFunctionTypes.WINDOW_CONDITION_FN,
+        ]:
+            raise ValueError(
+                "SparkDFExecutionEngine only supports map_condition_fn for column_pair_condition_partial partial_fn_type"
```

Comment:
From the above block, it looks like `window_condition_fn` is also supported.

Reply:
@fjork3 Great catch -- updated!  Thank you!